acceptance-tests/src/main/kotlin/org/ostelco/at/common/StripePayment.kt

```diff
@@ -23,24 +23,32 @@ object StripePayment {
         return token.id
     }
 
-    fun createPaymentSourceId(): String {
+    fun getCardIdForTokenId(tokenId: String) : String {
 
         // https://stripe.com/docs/api/java#create_source
         Stripe.apiKey = System.getenv("STRIPE_API_KEY")
 
-        // TODO martin: set valid map values
-        val sourceMap = mapOf<String,Any>()
-        val source = Source.create(sourceMap)
-        return source.id
+        val token = Token.retrieve(tokenId)
+        return token.card.id
     }
 
-    fun getCardIdForTokenId(tokenId: String) : String {
+    fun getDefaultSourceForCustomer(customerId: String) : String {
```

Comment:
I think this function is already implemented in our API or do we only have setDefaultSource ?

Reply:
ok, I see we have implementation in org.ostelco.prime.paymentprocessor but not exposed through client API